anitab-org/bridge-in-tech-web
# Feat: View members list

### Description
Allow user to see list of members

Fixes #58 

### Type of Change:
- Code
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
- run the application
- with user not logged-in go to `Members` page
- see login form
- login as a user
- now go to `Members` page
- see members list. (**Note** make sure you have created at least 3 verified users and 1 non verified users. The list should contain the 2 verified users beside the logged-in user)

<img width="965" alt="Screen Shot 2020-07-30 at 11 11 37 pm" src="https://user-images.githubusercontent.com/29667122/88927113-4f099700-d2ba-11ea-8f16-2bc7b14f0356.png">



### Checklist:


**Code/Quality Assurance Only**
